beacon/sync/src/test/java/tech/pegasys/teku/beacon/sync/forward/multipeer/chains/ThrottlingSyncSourceTest.java

```diff
@@ -75,23 +106,33 @@ void shouldRequestBlocksImmediatelyIfRateLimitNotExceeded() {
     ignoreFuture(source.requestBlocksByRange(UInt64.valueOf(100), count, blocksListener));
 
     // Both requests happen immediately
-    ignoreFuture(verify(delegate).requestBlocksByRange(UInt64.ZERO, count, blocksListener));
-    ignoreFuture(verify(delegate).requestBlocksByRange(UInt64.valueOf(100), count, blocksListener));
+    ignoreFuture(
+        verify(delegate)
+            .requestBlocksByRange(
+                eq(UInt64.ZERO), eq(count), any(RpcResponseListenerWithCount.class)));
+    ignoreFuture(
+        verify(delegate)
+            .requestBlocksByRange(
+                eq(UInt64.valueOf(100)), eq(count), any(RpcResponseListenerWithCount.class)));
   }
 
   @Test
   void shouldRequestBlobSidecarsImmediatelyIfRateLimitNotExceeded() {
-    final UInt64 count = UInt64.valueOf(MAX_BLOB_SIDECARS_PER_MINUTE - 1);
+    // 100 / 6 = 16, 16 * 6 = 92 < 100
+    final UInt64 count = UInt64.valueOf(16);
     ignoreFuture(source.requestBlobSidecarsByRange(UInt64.ZERO, count, blobSidecarsListener));
     ignoreFuture(
-        source.requestBlobSidecarsByRange(UInt64.valueOf(100), count, blobSidecarsListener));
+        source.requestBlobSidecarsByRange(UInt64.valueOf(16), count, blobSidecarsListener));
```

Comment:
same here, i think we should be using count if we've computed it anyway... given this shows it changing so substantially we'll need to be super careful that we've fixed everywhere that uses `requestBlobSidecarsByRange`